src/Components/FileInput.js/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+import { v4 as uuidv4 } from 'uuid';
+
+import DrowImages from "./DrowImages";
+
+import style from "./index.module.scss";
+
+
+const FileInput = ({avatar, setPhotosToFB}) => {
```

Comment:
Please add spaces between curly brackets, if the object or destructor contains only one line.